src/analysis/RDFatomistic.py

```diff
@@ -1,46 +1,60 @@
-#  Copyright (C) 2012,2013
+#  Copyright (C) 2012,2013,2014,2015,2016
 #      Max Planck Institute for Polymer Research
 #  Copyright (C) 2008,2009,2010,2011
 #      Max-Planck-Institute for Polymer Research & Fraunhofer SCAI
-#  
+#
 #  This file is part of ESPResSo++.
-#  
+#
 #  ESPResSo++ is free software: you can redistribute it and/or modify
 #  it under the terms of the GNU General Public License as published by
 #  the Free Software Foundation, either version 3 of the License, or
 #  (at your option) any later version.
-#  
+#
 #  ESPResSo++ is distributed in the hope that it will be useful,
 #  but WITHOUT ANY WARRANTY; without even the implied warranty of
 #  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 #  GNU General Public License for more details.
-#  
+#
 #  You should have received a copy of the GNU General Public License
-#  along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 
 r"""
 ************************************
 **espressopp.analysis.RDFatomistic**
 ************************************
 
+Class to compute radial distribution functions in adaptive resolution simulations in subregions of the box.
 
-.. function:: espressopp.analysis.RDFatomistic(system, type1, type2, _span)
+On the one hand, the RDF can be calculated in a cuboid region in the center of the box (periodic in y,z, limited in x). In this case, particle pairs are considered for which at least one of them is in the defined cuboid region (spanbase = True). This can be useful when the high resolution region has a slab geometry. No further normalization should be required. On the other hand, the routine can also calculate unnormalized RDFs using particle pairs with both particles being in the high resolution region. This can be useful when atomistic region has complicated or spherical geometries. In any case, only atoms belonging to different molecules (i.e. different coarse-grained entities) are considered.
```

Comment:
How about: 

"OPTION 1 (SPANBASE=TRUE) : the RDF can be calculated in a cuboid region in the center of the box (periodic in y,z, limited in x). In this case, particle pairs are considered for which at least one of them is in the defined cuboid region. This can be useful when the high resolution region has a slab geometry. No further normalization should be required. 

OPTION 2 (SPANBASE = FALSE):  the routine can also calculate unnormalized RDFs using particle pairs with both particles being in the high resolution region. . This can be useful when atomistic region has complicated or spherical geometries. 

In any case, only PAIRS OF ATOMISTIC PARTICLES belonging to TWO different COARSE-GRAINED PARTICLES are considered." 

Did I understand correctly?